infra/nnfw/command/install

```diff
@@ -7,5 +7,5 @@ if [[ ! -d "${BUILD_PATH}" ]]; then
   exit 255
 fi
 
-cd ${BUILD_PATH}
-make install
+# Require cmake version >= 3.15
+cmake --install ${BUILD_PATH} "$@"
```

Comment:
I've tried to modify install path to %{buildroot} as we talked in offline, but it fails.
Too much files (e.g. test packages under %{test_install_path}) are installed to modified pathes.
Anyway, current modification just works without any errors. 👍